include/pressiodemoapps/swe2d.hpp

```diff
@@ -129,7 +124,7 @@ create_problem_eigen(const mesh_t & meshObj,
 		     CustomBCsFunctorFront && customBCsFront,
 		     CustomBCsFunctorRight && customBCsRight,
 		     CustomBCsFunctorBack && customBCsBack,
-		     [[maybe_unused]] int icFlag = 0 )
+		     const std::unordered_map<std::string, typename mesh_t::scalar_t> & paramsMap)
```

Comment:
Also, fairly certain this will break Schwarz with the removal of the initial condition flag (since it's still required for the 2D Euler problem class), but presumably that won't be an issue when 2D Euler gets similarly parameterized.

Reply:
the init cond flag is not removed, i kept the overload exactly for that reason

Reply:
The other overload doesn't permit setting the parameterization though, only setting the icFlag and then using default parameterization.

Reply:
yes because i kept it to avoid breaking the *current* impl you have. But we said we will change it once the euler stuff changes too right?

Reply:
something will break anyway until we have a similar parametrization for euler

Reply:
Ah gotcha, thanks.